scripts/screener/screener.types.ts

```diff
@@ -88,3 +88,10 @@ export type ScreenerTestsConfig = {
   steps?: ScreenerStep[];
   themes?: ScreenerThemeName[];
 };
+
+export interface ScreenerProxyPayload {
```

Comment:
our current screener proxy is just a node server app. It's possible to publish package for this from the ADO repo, but it would need to be setup to publish and then turn the screener-proxy repo into a monorepo.... I can create an issue for a follow up, but I would say not in the scope of current work.

Reply:
created #21655 to follow up and added it to the build board